src/api/config/routes/webui_routes.rb

```diff
@@ -123,6 +123,10 @@
       resources :branches, controller: 'webui/packages/branches', only: [:create], constraints: cons do
         get '/:project', action: :into, on: :new, as: :project, constraints: cons
       end
+      resource :change_devel_projects, controller: 'webui/requests/change_devel_projects', only: [] do
```

Comment:
If we put this under `resource :projects`, then under `resource :packages`, the custom route definitions aren't needed. We have this setup already on master, but I believe you'll have to rebase. It's around the line 267 in this file once you're rebased.

Reply:
Put under `resource :projects`.